bsb/storage/_files.py

```diff
@@ -466,7 +466,15 @@ class MorphologyDependencyNode(FilePipelineMixin, FileDependencyNode):
     """
 
     pipeline = config.list(type=MorphologyOperation)
-    name = config.attr()
+    name = config.attr(type=str, default=None, required=False)
+    """
+    Name associated to the morphology. If not provided, the program will use the name of the file 
+    in which the morphology is stored. 
+    """
+    tags = config.attr(type=dict, default=None, required=False)
```

Comment:
```suggestion
    tags = config.attr(type=types.dict(type=types.or_(str, types.list(str))))
```

Stricter types prevent user errors. Default `default` is `None` already.